### PR TITLE
Add aggregateOffer property to Product JSONLD

### PR DESCRIPTION
**Pull request by pueyo5**
## Description of Change(s):

Add `AggregateOffer` support.

As we can read on [Google documentation](https://developers.google.com/search/docs/data-types/product#product), the `offers`property of Product is `An nested Offer or AggregateOffer to sell the product`.

Now `ProductJsonLd` allows `offers` and `aggregateOffer`.

Only allows one at a time, if both are set, `aggregateOffer` is ignored.

---

Some things to help get a PR reviewed and merged faster:

- Have you updated the documentation to go with your changes? Yes
- Have you written or updated unit tests? Offers property is already set, then the aggregateOffer property is ignored.
- Have you written an integration test in the test app supplied? Yes

The above are generally required on all PR's.

Please have a read of the Contributing Guide for full details.

https://github.com/garmeeh/next-seo/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: e2e/pages/jsonld.tsx [hunk 1]
```diff
@@ -195,6 +195,12 @@ export default () => (
           },
         },
       ]}
+      aggregateOffer={{
+        priceCurrency: 'USD',
+        lowPrice: '119.99',
+        highPrice: '199.99',
```
Comment (wyeo) [inline e2e/pages/jsonld.tsx:201]: The `highPrice` is `139.99`

Comment (pueyo5) [reply e2e/pages/jsonld.tsx]: I used the data from google developers docs.
FIXED ✔️

**Comment by pueyo5** [comment]
> Is this still relevant? If so, what is blocking it? Is there anything you can do to help move it forward?
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

Still pending

**Comment by garmeeh** [comment]
@all-contributors add @pueyo5 for doc code

**Review by garmeeh — APPROVED** [review]
Thanks @pueyo5. Sorry about the delay.

**Pull request merged**